### DISCUSSION TITLE
Members Only Sites

### DISCUSSION THREAD

**Discussion by FLMMFL** [GetPublii/Publii#1922] [General]
Hi guys, amazing platform ever since I started using it and so quick. Love it. 

I want to create a members only area...or members only site ,  is that possible with publii

**Comment by candidexmedia** [comment]
There have been [talks about creating gated content in this forum](https://github.com/GetPublii/Publii/discussions/1494), and even a possible [demo paywall plugin floating around](https://p4p.pages.dev/plugin-paywall/) by @howtomeetladies in the works, but nothing concrete yet...

I was able to get [Authorizer.dev](https://authorizer.dev/) working with a Publii site locally a few years ago, but I didn't know how to connect it to a payment processor like Stripe. If you don't need the member's area to be premium, I'd look into that.

**Reply by candidexmedia** [reply]
Other possible solutions:

- Hyvor: https://talk.hyvor.com/docs/gated-content
- Memberspace: https://help.memberspace.com/category/364-custom-html-guides
- Memberstack: https://docs.memberstack.com/hc/en-us/articles/11217223809819-Vanilla-JS-Guide
- Userbase: https://userbase.com/
